trace_stats/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+project(trace_stats)
+
+set(CMAKE_CXX_STANDARD 17)
+
+
+if(MSVC)
+  set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} /MP")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+endif()
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+
+file(GLOB HDR_FILES "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
+file(GLOB SRC_FILES "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
+
+include_directories(${THIRDPARTY_DIRECTORY}/Vulkan-Headers/include
```

Comment:
i think it is recommended to use target_include_directories, something like:

target_include_directories(${PROJECT_NAME} PRIVATE
  ${THIRDPARTY_DIRECTORY}/Vulkan-Headers/include
  ${CMAKE_SOURCE_DIR}
  ${CMAKE_BINARY_DIR}
  ${LibArchive_INCLUDE_DIRS})